mon-pix/app/components/routes/campaigns/assessment/skill-review.js

```diff
@@ -49,6 +49,31 @@ export default class SkillReview extends Component {
     return this.args.model.campaignParticipation.campaignParticipationResult.get('stageCount');
   }
 
+  get showCustomResultPageDescription() {
+    if (this.args.model.campaignParticipation.campaign.get('customResultPageText')) {
+      return this.args.model.campaignParticipation.campaign.get('customResultPageText').length > 0;
+    }
+    return false;
+  }
+
+  get customResultPageDescription() {
+    return this.args.model.campaignParticipation.campaign.get('customResultPageText');
+  }
+
+  get hasOrganizationLogo() {
+    if (this.args.model.campaignParticipation.campaign.get('organizationLogoUrl'))
+      return this.args.model.campaignParticipation.campaign.get('organizationLogoUrl').length > 0;
+    return false;
+  }
```

Comment:
**suggestion** : ici on a de la logique inutile, en JS, si l'attribut `organizationLogoUrl` est rempli, alors c'est truthy, du coup on peut faire directement dans le hbs
`{{#if this.organizationLogoUrl}}`